recipes/tsil/all/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+cmake_minimum_required(VERSION 3.4)
+project(TSIL C)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
```

Comment:
useless

Reply:
You are right. I've removed it in commit d4ca865d